src/Init/Data/List/Basic.lean

```diff
@@ -164,13 +170,19 @@ theorem isEqv_cons₂ : isEqv (a::as) (b::bs) eqv = (eqv a b && isEqv as bs eqv)
 
 /-! ## Lexicographic ordering -/
 
-/-- Lexicographic ordering for lists. -/
+/--
+Lexicographic ordering for lists with respect to an ordering of elements.
+-/
 inductive Lex (r : α → α → Prop) : List α → List α → Prop
   /-- `[]` is the smallest element in the order. -/
   | nil {a l} : Lex r [] (a :: l)
-  /-- If `a` is indistinguishable from `b` and `as < bs`, then `a::as < b::bs`. -/
+  /--
+  Two lists with the same head, but ordered tails, are ordered.
```

Comment:
```suggestion
  Two lists with the same head, and ordered tails, are ordered.
```
"ordered tails" sounds to me like the tails are *sorted*, rather than one being LE the other.

Reply:
I opted for a bigger rephrase than this suggestion:
```
  Two lists with the same head are lexicographically ordered if their tails are lexicographically
  ordered.
```
Thanks!

Reply:
Actually, for consistency with the other update:

> If two lists have the same head, then their tails determine their lexicographic order. If the tail of the first list is lexicographically smaller than the tail of the second list, then the entire first list is lexicographically smaller than the entire second list.